### PR TITLE
Fixed error when there are no annotations in vmrk file

### PR DESCRIPTION
**Pull request by akovrig**
Before this fix, attempting to access a vhdr file with associated vmrk would give the following error:
```
~mne-python/mne/io/brainvision/brainvision.py in _read_annotations_brainvision(fname, sfreq)
    257         The annotations present in the file.
    258     """
--> 259     onset, duration, description, date_str = _read_vmrk(fname)
    260     orig_time = _str_to_meas_date(date_str)
    261 

ValueError: not enough values to unpack (expected 4, got 0)
```
Presumably because there were no annotations. Now the code skips attempting to retrieve annotations when there are none.

### CODE REVIEW

**Comment by jasmainak** [comment]
Can you add a test?

**Comment by larsoner** [comment]
This pull request **introduces 5 alerts** when merging 7aa9ff780fc0a2da159d51ea96b9efd745aa8131 into d0b8c005d3e14f911c6615788e4428052cbe2819 - [view on LGTM.com](https://lgtm.com/projects/g/mne-tools/mne-python/rev/pr-d649d472dcd7d18da1feaf25403e8909b678bf7b)

**new alerts:**

* 5 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: mne/realtime/lsl_client.py [hunk 1]
```diff
@@ -83,8 +83,18 @@ def iter_raw_buffers(self):
...
         # To use this function with an LSL stream which has a 'name' but no
         # 'source_id', change the keyword in pylsl.resolve_byprop accordingly.
+=======
+        """To use this function with an LSL stream which has a 'name' but no 
+        'source_id', change the keyword in pylsl.resolve_byprop accordingly."""
+>>>>>>> Comment use of keywords in pylsl.resolve_byprop.
+=======
+        # To use this function with an LSL stream which has a 'name' but no
+        # 'source_id', change the keyword in pylsl.resolve_byprop accordingly.
+>>>>>>> STY: Comments to please the pedantic CIs
```
Comment (agramfort) [inline mne/realtime/lsl_client.py]: @akovrig you need to fix your merge conflicts. Maybe start from a clean master and reapply your fix?
let us know if you need help. thanks

Comment (akovrig) [reply mne/realtime/lsl_client.py]: Here you go: https://github.com/mne-tools/mne-python/pull/6217

**Pull request closed without merging**